recipes/cuda-nvml-dev/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "cuda-nvml-dev" %}
+{% set version = "12.0.76" %}
+{% set cuda_version = "12.0" %}
+{% set platform = "linux-x86_64" %}  # [linux64]
+{% set platform = "linux-ppc64le" %}  # [ppc64le]
+{% set platform = "linux-sbsa" %}  # [aarch64]
+{% set platform = "windows-x86_64" %}  # [win]
+{% set target_name = "x86_64-linux" %}  # [linux64]
+{% set target_name = "ppc64le-linux" %}  # [ppc64le]
+{% set target_name = "sbsa-linux" %}  # [aarch64]
+{% set target_name = "x64" %}  # [win]
+{% set extension = "tar.xz" %}  # [not win]
+{% set extension = "zip" %}  # [win]
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/cuda/redist/cuda_nvml_dev/{{ platform }}/cuda_nvml_dev-{{ platform }}-{{ version }}-archive.{{ extension }}
+  sha256: 6780e554784bef73050144afe39668e26923d2533c268460c348e0e380d5d048  # [linux64]
+  sha256: 319b96999d144192094d254f0607debb14118ce206ef7dfeba3d13da1cc805ec  # [ppc64le]
+  sha256: 4aee09e48ecd30b2dcaa9de4eb8af2fcdf36ed040eac50c14f3345a12410b6ec  # [aarch64]
+  sha256: abc366280d0aa1ca047323a3e316270b80ffabb649021184647cd16b44c60a71  # [win]
+
+build:
+  number: 0
+  skip: true  # [osx]
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
+    - arm-variant * {{ arm_variant_type }}  # [aarch64]
+  host:
+    - cuda-version {{ cuda_version }}
+  run:
+    - {{ pin_compatible("cuda-version", max_pin="x.x") }}
+  run_constrained:
+    - arm-variant * {{ arm_variant_type }}  # [aarch64]
+
+test:
+  commands:
+    - test -f $PREFIX/targets/{{ target_name }}/lib/stubs/libnvidia-ml.so  # [linux]
+    - test -f $PREFIX/targets/{{ target_name }}/include/nvml.h             # [linux]
```

Comment:
Should this land in `$PREFIX/include` as well?

Reply:
We held off on putting symlinks in `$PREFIX/include` for all packages. IIRC there were concerns with the generic naming of headers, such as `math_functions.h` and others. Using `nvcc` will include the `targets` directory by default.

Reply:
Fair enough. I think we should prioritize a strategy around this because most of these are host functions where `nvcc` isn't required and generally shouldn't be used if there's no kernels / device code.